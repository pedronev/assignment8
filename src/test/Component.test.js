import React from 'react';
import EnquiryList from '../components/enquiryList';
import {create} from 'react-test-renderer';

describe('EnquiryList Component Snapshot', () => {
    test("EnquiryList",() => {
        let tree = create(<EnquiryList/>);
        expect(tree.toJSON()).toMatchSnapshot();
    })
})
