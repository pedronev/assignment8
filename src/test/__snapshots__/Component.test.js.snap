// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EnquiryList Component Snapshot EnquiryList 1`] = `
<div>
  <div />
</div>
`;
